ansible_base/models/organization.py

```diff
@@ -0,0 +1,30 @@
+"""Organization models."""
+from django.conf import settings
+from django.db import models
+
+from .common import UniqueNamedCommonModel
+
+
+class AbstractOrganization(UniqueNamedCommonModel):
+    """An abstract base class for organizations."""
+
+    class Meta:
+        abstract = True
+
+    description = models.TextField(
+        null=False,
+        default="",
+        help_text="The organization description.",
+    )
+
+    users = models.ManyToManyField(
```

Comment:
This is starting to be somewhat duplicated work with https://github.com/ansible/aap-gateway/pull/164

At least you and @relrod seem to be on the same page about the name of this field.

Reply:
And it will be even more duplication, since I'm working on organizations in Hub.